changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+# Unreleased
+
+### Added
+- `edsnlp/tune.py`: New hyperparameters tuning script.
+- `tests/tuning/test_tuning.py`: Unit tests for the tuning script.
+- `docs/tutorials/tuning.md`: New tutorial for hyperparameter tuning.
```

Comment:
Can you instead explain the new feature, since the changelog is more user and feature-oriented ?